### PR TITLE
Added proposal for a simple email signature.

### PR DESCRIPTION
**Pull request by Ainali**
After our brief discussion, and after have added one myself, this section might be of use for others.

-----
[View rendered activities/communication/index.md](https://github.com/publiccodenet/about/blob/activities-communication-email-signature/activities/communication/index.md)

### CODE REVIEW

**Comment by bvhme** [comment]
Hey hey,

I also need a phone number in there, perhaps also a mobile, and Twitter.

Here is my current signature:

> ​--
> Boris van Hoytema, chief executive and secretary
> boris@publiccode.net – +31 20 2 444 555 – @bvhme

I personally prefer more compact signatures, but longer is also all right.

**Comment by Ainali** [comment]
Agreed that phone and Twitter might be useful. But the email would be redundant since the signature is used when sending email from that address, wouldn't it?

File: activities/communication/index.md [hunk 1]
```diff
@@ -20,3 +20,16 @@ Phone numbers and lines are managed in our VOIP supplier Voys' [online managemen
...
+In order to clearly convey who is communicating, this template for email signature could be used. It has no styling in order to make it more probable that email clients and mailing lists will handle the signature gracefully.
+
+Name, Role  
+Email | Phone | Twitter 
+Foundation for Public Code https://publiccode.net  
+
+*Example:*
+Boris van Hoytema, Chief Executive and secretary
+boris@publiccode.net | +31 20 2 444 555 | @bvhme
+Foundation for Public Code https://publiccode.net
```
Comment (ElenaFdR) [inline activities/communication/index.md]:
```suggestion
[Foundation for Public Code](https://publiccode.net/) 
```
The lack of punctuation here bothers me. I suggest we hyperlink the name.

Comment (Ainali) [reply activities/communication/index.md]: @ElenaFdR  If we were to not use any markup in the footer, what sort of punctuation would make you less bothered?

**Comment by ericherman** [comment]
Based upon the above template, I personalized mine as:
-- 
Eric Herman, Lead codebase steward for quality
Foundation for Public Code | https://publiccode.net
github.com/publiccodenet | github.com/ericherman
eric@publiccode.net | +31 620719662 | @Eric_Herman

Which contains all things, and displays okay on narrow mobile readers

**Pull request merged**